### DISCUSSION TITLE
Can't change workspaces if they do not contain a number

### DISCUSSION THREAD

**Discussion by bbommarito** [regolith-linux/regolith-desktop#691] [Q&A]
This is an odd issue that I cannot seem to track down. I've gone through all the default configs and can't see a reason for this behavior, but perhaps I am overlooking something.

I like having my workspace names either be symbols, or simple words. I don't need to see a number because my workspaces have been in the same order forever. However, that doesn't seem possible with Regolith's stock configs.

If I do something like this in my `~/config/regolith2/Xresources` file:
`i3-wm.workspace.01.name: "term"`
I can never switch to workspace 1 when using `Super+1`

If, however, I do this:

`i3-wm.workspace.01.name: "1 term"`
I can switch to the workspace using `Super+1`
The `1` will not display, instead what I see is ` term`

If I do something like this:
`i3-wm.workspace.01.name: "1 1.) term"`
Then I see: ` 1.) term`

All other configs are stock.

My questions are: A.) Am I just broken (likely, yes)? B.) Is there a config I'm missing that is causing this odd behavior? C.) Is this related to i3 itself?

I am using Regolith 2.1 and I am logging out between each change because I know i3 is a bit weird with reloads.

**Comment by kgilmer** [comment]
Hi @burritocodes , in this regard Regolith does not vary from stock i3.  Workspace indexes in i3 utilize the "label" of the workspace to determine the index.  Specifically, the label string requires the prefix in a specific form so that i3 can parse the index from it.  This behavior is documented here: https://i3wm.org/docs/userguide.html#_strip_workspace_numbers_name.  I believe your goal is achievable if you conform to the i3/i3bar spec as linked.
